code/modules/vehicles/mecha/combat/gygax.dm

```diff
@@ -104,3 +104,17 @@
 		cell = C
 		return
 	cell = new /obj/item/stock_parts/cell/bluespace(src)
+
+/obj/vehicle/sealed/mecha/gygax/dark/add_scanmod(obj/item/stock_parts/scanning_module/sm=null)
```

Comment:
I don't see why you need to set the default value to null here, that should be implied

Reply:
Done